FCC Survey Page/css/sp-css.css

```diff
@@ -0,0 +1,113 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans+JP|Roboto+Slab&display=swap');
+
+html,
+body {
+    background-color: #A9D7D1;
+}
+
+#form-outer {
```

Comment:
**Recommended:** Put media query for mobile devices. Make the max width = 90/95%.